### ISSUE TITLE
Error with Fire and charcoal/Smelter 

### ISSUE THREAD

**Issue by Cxboxg** [abearcodes/Valheim#68]
[Error  : Unity Log] NotImplementedException: stub
Stack trace:
ABearCodes.Valheim.CraftingWithContainers.Smelting.SmelterPatch.SmelterOnAddOreAreaSearch (Smelter __instance, Switch sw, Humanoid user, ItemDrop+ItemData item, ZNetView ___m_nview, System.Boolean& __result) (at <697b90d1b592472e8017cb3be328c588>:0)
(wrapper dynamic-method) Smelter.DMD<Smelter::OnAddOre>(Smelter,Switch,Humanoid,ItemDrop/ItemData)
Switch.Interact (Humanoid character, System.Boolean hold, System.Boolean alt) (at <dcb0de0155834d459f18d7ac6ea5244b>:0)
Player.Interact (UnityEngine.GameObject go, System.Boolean hold, System.Boolean alt) (at <dcb0de0155834d459f18d7ac6ea5244b>:0)
(wrapper dynamic-method) Player.DMD<Player::Update>(Player)

pls help me

**Comment by cCaptainn** [comment]
Same issue. Seems to be specific to the smelters/fires

**Comment by WanderingCanuc** [comment]
Same problem. Only related to smelters / fires. Boxes and chests seem to still work.

**Comment by piinoCChio** [comment]
yup same bug here. definitely has sth to do with the latest valheim patch.

**Comment by lowend15** [comment]
Same issue noticed starting yesterday.  Thank you.

**Comment by SauceyGit** [comment]
+1 on this. The charcoal kiln will not accept wood, and the smelter will not accept ore (but _will_ accept coal) from containers or from the character's inventory. In order to do smelting or to use the kiln, the mod has to be disabled.

**Comment by jeffgentry** [comment]
Same issue with windmills and spinning wheels. Even if you have barley/flax on your person, you can't put barley in windmills or flax into spinning wheels. As previously noted, adding coal to a smelter or forge works as always.

**Comment by mattiaperry95** [comment]
same here, for me the fire works fine, but kiln and smelter dont work (same thing as the others)

**Comment by andersonimes** [comment]
Is this mod being maintained still? It'd be great to get a fix for this - this is my favorite mod by far, but I've had to disable it.

**Comment by cCaptainn** [comment]
> Is this mod being maintained still? It'd be great to get a fix for this - this is my favorite mod by far, but I've had to disable it.

I don't believe it is. I've moved to another mod with a very similar function (and more configs from what I can tell) https://valheim.thunderstore.io/package/frenchiveruti/Craft_From_Containers_by_aedenthorn/

**Comment by jeffgentry** [comment]
> 

I've moved to the same mod. Works great.
